etc/DockerHelper.sh

```diff
@@ -157,10 +160,14 @@ _push() {
                     2>&1 | tee build/create-ubuntu20-latest.log
                 ./etc/DockerHelper.sh create -target=dev -os=ubuntu20 -sha \
                     2>&1 | tee build/create-ubuntu20-${commitSha}.log
-                ./etc/DockerHelper.sh create -target=dev -os=ubuntu22 \
-                    2>&1 | tee build/create-ubuntu22-latest.log
-                ./etc/DockerHelper.sh create -target=dev -os=ubuntu22 -sha \
-                    2>&1 | tee build/create-ubuntu22-${commitSha}.log
+                ./etc/DockerHelper.sh create -target=dev -os=ubuntu22.04 \
+                    2>&1 | tee build/create-ubuntu22.04-latest.log
+                ./etc/DockerHelper.sh create -target=dev -os=ubuntu22.04 -sha \
+                    2>&1 | tee build/create-ubuntu22.04-${commitSha}.log
+                ./etc/DockerHelper.sh create -target=dev -os=ubuntu22.10 \
+                    2>&1 | tee build/create-ubuntu22.10-latest.log
+                ./etc/DockerHelper.sh create -target=dev -os=ubuntu22.10 -sha \
+                    2>&1 | tee build/create-ubuntu22.10-${commitSha}.log
```

Comment:
Could you change these repetitive commands to be in a for loop? Below as well for the `test` and dry-run push.